fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/SparkDpp.java

```diff
@@ -507,17 +558,6 @@ private void processRollupTree(RollupTreeNode rootNode,
             dataframe = dataframe.withColumn(mappingColumn,
                     functions.expr(mappingDescription).cast(dstTableSchema.apply(mappingColumn).dataType()));
         }
-        // projection and reorder the columns
-        dataframe.createOrReplaceTempView("src_table");
-        StringBuilder selectSqlBuilder = new StringBuilder();
-        selectSqlBuilder.append("select ");
-        for (String name : dstColumnNames) {
-            selectSqlBuilder.append(name + ",");
-        }
-        selectSqlBuilder.deleteCharAt(selectSqlBuilder.length() - 1);
-        selectSqlBuilder.append(" from src_table");
-        String selectSql = selectSqlBuilder.toString();
-        dataframe = spark.sql(selectSql);
```

Comment:
Don't delete. Source data schema may be different from load table schema.
These code is used for projection and reorder columns according to columns order in table schema.